src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -10,7 +10,13 @@ Looking for instructions to set up the Node serverless SDK manually? [Check out
 
 </Alert>
 
-Connect Sentry to your AWS account to automatically add Sentry error and performance monitoring to your Node Lambda functions.
+<Alert title="Note" level="info">
+
+Looking for instructions to set up the Python serverless SDK manually? [Check out these docs instead](/platforms/python/guides/aws-lambda/).
+
+</Alert>
```

Comment:
Let's combine these two, and change their structure. Use the "Note and /Note" approach